AUTHORS.md

```diff
@@ -4,4 +4,5 @@ The Image Map module would not be possible without the generous contributions of
 
 We specifically want to thank our developers Kyle Chesney kyle.chesney@ufl.edu, James Pence jrpence@ufl.edu, and Philip Chase pbc@ufl.edu for their contributions to the project.  Taryn Stoffs tls@ufl.edu and Jerri Johnston j.johnston@ufl.edu provided valuable input into the design and testing of the Image Map module.
 
-Geoffroey-Allen Franklin frgeof Provided the new image and mapping for Pirads v2.1.
+Geoffroey-Allen Franklin gfranklin@atsu.edu Provided the new image and mapping for Pirads v2.1.
+Jane Coe Johnson jjohnson@atsu.edu assisted Geoffroey-Allen updating information about the DO-Touch.NET image maps.
```

Comment:
I'm actually not sure if we should list people here since there's a precedence for image contributions being in the README